distributed/deploy/ssh.py

```diff
@@ -72,16 +70,23 @@ def __init__(
         address: str,
         connect_options: dict,
         kwargs: dict,
-        worker_module="distributed.cli.dask_worker",
+        worker_module="deprecated",
+        worker_class="distributed.Nanny",
         remote_python=None,
         loop=None,
         name=None,
     ):
         super().__init__()
 
+        if worker_module != "deprecated":
+            raise ValueError(
```

Comment:
Should this be a `DeprecationError`?

Reply:
I could change this to a `DeprecationWarning`, though I'm not sure if it would confuse users to get a warning that interrupts the program. In the original PR, it seemed like @mrocklin [preferred ](https://github.com/dask/distributed/pull/3907#discussion_r440941318) raising an explicit error rather than a warning so that we wouldn't have to support both `worker_module` and `worker_class`.

From that discussion, it seems like the main motive behind `worker_module` was for dask-cuda benchmarks, so once this and rapidsai/dask-cuda#326 are merged we should be able to remove this error and the `worker_module` kwarg altogether (though we'll probably want to wait a bit to see if there are any downstream breakages from this).

Reply:
Fair enough.